content/account/organizations.textile

```diff
@@ -0,0 +1,193 @@
+---
+title: Ably Organizations
+meta_description: "Manage Ably organizations, provision users, configure SSO with SCIM and Okta, and handle account roles and permissions."
+meta_keywords: "Ably organizations, user provisioning, SSO, SCIM, Okta, access management, Ably dashboard, account roles, permissions"
+---
+
+Use Ably Organizations to efficiently manage multiple Ably accounts by centralizing user access and permissions under a single organizational structure. With Ably Organizations, you streamline user provisioning through a single Sign-On (SSO) configuration and utilize System for Cross-domain Identity Management (SCIM) for group-based access control. This allows you to assign roles and permissions across different accounts—such as production, staging, and development environments—while maintaining isolated capacities and billing for each.
+
+<aside data-type='note'>
+<p>To start using organizations, an Enterprise account is required. "Contact":https://ably.com/contact Ably to migrate to an Enterprise account. </p>
+</aside>
+
+h2(#primary). Primary account
+
+The primary account is an organization's main account and holds the following privileges:
+
+* The highest level of access to the organization.
+* Ownership of all all accounts within the organization.
+* The default account used for provisioning.
+
+<aside data-type='note'>
+<p>To change the primary account "contact":https://ably.com/contact Ably.</p>
+</aside>
+
+h2. Create accounts in an organization
+
+Create accounts in an organization:
+
+* Open the *Account* navigation dropdown.
+** Click *Organization Accounts*.
+** Click *New account*.
+** Add an account name and *Create account*.
+
+h2. Isolate environments
+
+Use separate accounts within an organization to create isolated environments, such as production, staging, and development. Assign each environment a "package":/docs/pricing#packages that meets its specific needs. For example, production may need high capacity with an *Enterprise* package, while staging might use a *Standard* package. Development may operate on a *Free* package.
+
+Select the appropriate package for your account:
+
+* Open the *Account* navigation dropdown.
+** Click on *Billing*.
+** Choose the required package for the selected account and *Save billing details*.
+
+<aside data-type='note'>
+<p>Each account within an Ably Organization is assigned a unique domain. This ensures that environments such as production, staging, and development are properly isolated from one another. </p>
+</aside>
+
+h2. Manage user account access
+
+Manage user account access:
+
+* Open the *Account* navigation dropdown.
+** Click *Organization Users*.
+** Select the user's account access you want to manage.
```

Comment:
It's hard to tell if this is choosing an account they should have access to, or the user themselves.